index.ts

```diff
@@ -12,6 +12,8 @@ const network =
     ? Network.DEVNET
     : Network.LOCALNET;
 
+export let fetchingMarkets = new Map<assets.Asset, boolean[]>();
```

Comment:
is this syntax right?? runs locally but i feel like ive never seen `export let` before lol